src/components/DataListContainer.vue

```diff
@@ -1,10 +1,110 @@
-<template></template>
+<template>
+  <v-container fluid>
+    <v-data-iterator
+      :items="items"
+      :items-per-page.sync="itemsPerPage"
+      :footer-props="{ itemsPerPageOptions }"
+    >
+      <template v-slot:default="props">
+        <template v-for="(item, index) in props.items">
+          <component
+            :is="getChildComponent(item.child)"
+            v-bind="item.props"
+            :key="index">
+          </component>
+        </template>
+      </template>
+    </v-data-iterator>
+  </v-container>
+</template>
 
 <script>
+import QuestionListItem from './QuestionListItem';
+
+const dummyData = [
+  {
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+    },
+  },
+  {
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+    },
+  },
+  {
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+      subCategory: 'Some categ',
+    },
+  },
+  {
+
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+    },
+  },
+  {
+
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+      subCategory: 'Some categ',
+    },
+  },
+  {
+
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+      subCategory: 'Some categ',
+    },
+  },
+  {
+
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+    },
+  },
+  {
+
+    props: {
+      title: 'Frozen Yogurt',
+      category: 'Coding',
+      tags: ['some', 'random', 'tag'],
+      subCategory: 'Some categ',
+    },
+  },
+];
+
 export default {
-  name: "DataListContainer"
+  name: 'DataListContainer',
+  data: () => ({
+    childComponent: QuestionListItem,
+    itemsPerPageOptions: [4, 8, 12],
+    itemsPerPage: 4,
+    items: dummyData,
```

Comment:
@pbteja1998 `childComponent` expects a component which is used to render the list items. `dummyData` contains the props that are expected by the `childComponent`. In addition, you may want some children to be of different component then you can add something like this:
```javascript
//Assume that card is imported already.
const dummyData = [
...,
{
    child: Card,
    props: {
      title: 'Frozen Yogurt',
      category: 'Coding',
      tags: ['some', 'random', 'tag'],
      //props that are expected by the Card component.
    },
  },
...,
```